pages/about.html

```diff
@@ -4,9 +4,62 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link rel="stylesheet" href="style.css">
+  <title>About me</title>
 </head>
 <body>
-  
+  <div class="container">
+    <header>
+      <h1>My life </h1>
+    <nav>
+        <ul>
+          <div class="box">
+          <div><li><a class="btn" href="/pages/about.html">About me</a></li></div>
+          <div><li><a class="btn" href="/pages/portfolio.html">Portfolio</a></li></div>
+          <div><li><a class="btn" href="/pages/index.html">Home</a></li></div>
+          </div>
+        </ul>
+    </nav>
+  </header>
+      
+    <main>
+      <section>
+        <h2>My life in Colombia</h2>
+        <p>
+          I was raised and born in Medellin, Colombia one of the best cities I have seen. I grew up being independent, but also very close to my family. 
+          I spent most of my time going to the gym and riding my motorcycle. 
+          I have been always very curios about life, and love learning new things every day.    
+        </p>
+        <img class="about" alt="Colombia" src="/images/Family.png"/>
```

Comment:
It would be better if the alt text is more descriptive. Screen readers use the alt text to tell people who are blind what is in the picture. Just hearing "Colombia" does not really explain to the blind user what is in the picture.